src/utils.ts

```diff
@@ -208,7 +208,7 @@ export function isDynamicPattern(pattern: string, options?: { caseSensitiveMatch
 
 // #region log
 export function log(...tasks: unknown[]): void {
-  console.log(`[tinyglobby ${new Date().toLocaleTimeString('es')}]`, ...tasks);
+  console.log(`[tinyglobby ${new Date().toLocaleTimeString()}]`, ...tasks);
```

Comment:
reason why it was set to spanish was so that it used the `HH:mm:ss` format, so that debug logs look the same regardless of region